Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:latest
+
+# Create the docs website directory
+WORKDIR /website
+
+COPY . .
+
+WORKDIR /website/website
```

Comment:
That is weird. Do we need this structure or can we flatten it?

Reply:
I tried to keep the project's folder structure (as the original project also has a weird /website/website hierarchy). The outer folder can be renamed to any other name (maybe "verdaccio.org"?).
Maybe the original folder structure should be changed on the repository as well? @DanielRuf

Reply:
The first level is the name of the repo after a default git clone but you can proide a different directory name as additional parameter.

This is because the website was split fro the original repository and has this structure now.

Reply:
> verdaccio.org

Better not a dot org, why not simply `verdaccio-website`?

Reply:
changed 👍